packaging/docker/kubernetes/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# Dockerfile
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2021 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This docker image assumes that the context for the docker build is pointed
+# at the root of the foundationdb repository.
+
+# Build the Kubernetes monitor
+
+FROM golang:1.16.7-bullseye AS go-build
```

Comment:
This pattern is foreign to our current build process. What is `golang:1.16.7-bullseye` based on? Does it matter?

Reply:
bullseye is Deebian and it shouldn't matter.